crates/vision/src/ball_detection.rs

```diff
@@ -1,19 +1,12 @@
 use color_eyre::Result;
 use context_attribute::context;
 use framework::{AdditionalOutput, MainOutput};
-use types::{
-    configuration::BallDetection as BallDetectionConfiguration, Ball, CameraMatrix,
-    CandidateEvaluation, PerspectiveGridCandidates,
-};
+use types::{Ball, CameraMatrix, CandidateEvaluation, PerspectiveGridCandidates};
 
 pub struct BallDetection {}
 
 #[context]
-pub struct CreationContext {
-    // TODO: remove these configuration structs and explicitely list parameters
-    pub ball_detection: Parameter<BallDetectionConfiguration, "ball_detection.$cycler_instance">,
-    pub ball_radius: Parameter<f32, "field_dimensions.ball_radius">,
-}
+pub struct CreationContext {}
```

Comment:
Instead of removing, maybe only comment them out? Then you think about them again when porting vision. Same for the other vision nodes.

Reply:
I disagree. Let's keep it like that, I already started porting vision